lib/Partitioner/Partitioner.cpp

```diff
@@ -94,8 +96,10 @@ llvm::Error Partitioner::finalize(const DAGListTy &partitions,
   if (dumpPartition) {
     for (const auto &node : partitions[0].nodes) {
       Function *subF = module_->getFunction(node->name);
-      RETURN_ERR_IF_NOT(
-          subF, strFormat("Invalid function name %s.", node->name.data()));
+      if (!subF) {
+        return MAKE_ERR(GlowErr::ErrorCode::PARTITIONER_ERROR,
+                        "Invalid function name" + node->name);
```

Comment:
space after "name"
Also same comment as above and why not continue to use strFormat?

Reply:
This one looks simpler :)